recipes/codes-ui/meta.yaml

```
@@ -0,0 +1,56 @@
+{% set metview_version = "5.5.0" %}
+{% set codes_ui_version = "1.2.3" %}
+
+package:
+  name: codes-ui
+  version: {{ codes_ui_version }}
+
+source:
+  fn: Metview-{{ metview_version }}-Source.tar.gz
+  url: https://confluence.ecmwf.int/download/attachments/3964985/Metview-{{ metview_version }}-Source.tar.gz
+  sha256: 3841b097de7bc68e467a0cf7b829cdd0ac4485768eec9ae6426865ce56a48652
+
+build:
+  number: 1000
+  skip: True  # [win]
+  detect_binary_files_with_prefix: true
+
+requirements:
+  build:
+    - cmake
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ cdt('mesa-libgl-devel') }}  # [linux]
+    - {{ cdt('mesa-dri-drivers') }}  # [linux]
+    - {{ cdt('libselinux') }}  # [linux]
+    - {{ cdt('libxdamage') }}  # [linux]
+    - {{ cdt('libxfixes') }}  # [linux]
+    - {{ cdt('libxxf86vm') }}  # [linux]
+    - {{ cdt('libxcb') }}  # [linux]
+    - {{ cdt('libxext') }}  # [linux]
+  host:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - eccodes
+    - magics
+    - qt
+  run:
+    - eccodes
+    - magics
+    - qt
+
+test:
+  commands:
+    - conda inspect linkages -p $PREFIX codes-ui
+    - conda inspect objects -p $PREFIX codes-ui  # [osx]
+
+about:
+  home: https://confluence.ecmwf.int/display/METV/CodesUI
+  license: Apache 2.0
+  license_family: Apache
```

I'm not certain if the Apache 2.0 license requires it, but you should probably include it in the package using:
```suggestion
  license_family: Apache
  license_file: metview/src/codes_ui/LICENSE
```

---

Didn't know you could do this. Thanks :)